docker_entrypoint.sh

```diff
@@ -80,6 +80,13 @@ fi
 mkdir -p /root/.lightning/shared
 mkdir -p /root/.lightning/public
 
+# Don't backup lightningd DB
+if ! [ -e /root/.lightning/.backupignore ]; then
```

Comment:
You could also just have a `.backupignore` file in the repo and copy it over in the Dockerfile. If that was not possible or causing issue, this works.

Reply:
Refactored to copy backupignore in Dockerfile instead of creating it at runtime